variants/meshlink/variant.h

```diff
@@ -0,0 +1,233 @@
+  License along with this library; if not, write to the Free Software
+  Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+*/
+
+#ifndef _VARIANT_MESHLINK_
+#define _VARIANT_MESHLINK_
+#ifndef MESHLINK
+#define MESHLINK
+#endif
+/** Master clock frequency */
+#define VARIANT_MCK (64000000ul)
+
+//#define USE_LFXO // Board uses 32khz crystal for LF
+ #define USE_LFRC    // Board uses RC for LF
+
+/*----------------------------------------------------------------------------
+ *        Headers
+ *----------------------------------------------------------------------------*/
+
+#include "WVariant.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif // __cplusplus
+
+// Number of pins defined in PinDescription array
+#define PINS_COUNT (48)
+#define NUM_DIGITAL_PINS (48)
+#define NUM_ANALOG_INPUTS (2)
+#define NUM_ANALOG_OUTPUTS (0)
+
+
+
+#define BUTTON_PIN (38) // If defined, this will be used for user button presses,
+#define BUTTON_NEED_PULLUP
+
+
+// LEDs 
+#define PIN_LED1 (24) //24
+#define PIN_LED2 (-1)  //Watchdog enable pin25
+//#define PIN_LED3 (-1)
+
+//#define LED_RED PIN_LED1
+//#define LED_GREEN LED_PIN
+#define LED_BLUE PIN_LED1
+
+//#define LED_CONN PIN_LED1
+#define LED_BUILTIN PIN_LED1
+
+#define LED_STATE_ON 0 // State when LED is litted
+#define LED_INVERTED 1
+// Testing USB detection
+//#define NRF_APM
+/*
+ * Analog pins
+ */
+#define PIN_A0 (2)
+#define PIN_A7 (31)
+/*
+static const uint8_t A0 = PIN_A0;
+static const uint8_t A1 = PIN_A1;
+static const uint8_t A2 = PIN_A2;
+static const uint8_t A3 = PIN_A3;
+static const uint8_t A4 = PIN_A4;
+static const uint8_t A5 = PIN_A5;
+static const uint8_t A6 = PIN_A6;
+static const uint8_t A7 = PIN_A7;
+*/
+#define ADC_RESOLUTION 14
+
+// Other pins
+//#define PIN_AREF (2)
+
+
+//static const uint8_t AREF = PIN_AREF;
+
+/*
+ * Serial interfaces
+ */
+#define PIN_SERIAL1_RX (40)
+#define PIN_SERIAL1_TX (7)
+
+
+
+
+/*
+ * SPI Interfaces
+ */
+#define SPI_INTERFACES_COUNT 2
+
+#define PIN_SPI_MISO (8) 
+#define PIN_SPI_MOSI (32 + 9) 
+#define PIN_SPI_SCK (11)  
+
+#define PIN_SPI1_MISO (23)
+#define PIN_SPI1_MOSI (21)
+#define PIN_SPI1_SCK (19)
+
+static const uint8_t SS = 12;
+static const uint8_t MOSI = PIN_SPI_MOSI;
+static const uint8_t MISO = PIN_SPI_MISO;
+static const uint8_t SCK = PIN_SPI_SCK;
+
+/*
+ * eink display pins
+ */
+//#define USE_EINK
+
+#define PIN_EINK_CS (15)
+#define PIN_EINK_BUSY (16)
+#define PIN_EINK_DC (14)
+#define PIN_EINK_RES (17) //17
+#define PIN_EINK_SCLK (19)
+#define PIN_EINK_MOSI (21) // also called SDI
+
+
+
+/*
+ * Wire Interfaces
+ */
+#define WIRE_INTERFACES_COUNT 1
+
+#define PIN_WIRE_SDA (1)
+#define PIN_WIRE_SCL (27)
+
+// QSPI Pins
+#define PIN_QSPI_SCK 19
+#define PIN_QSPI_CS 22
+#define PIN_QSPI_IO0 21
+#define PIN_QSPI_IO1 23
+#define PIN_QSPI_IO2 32
+#define PIN_QSPI_IO3 20
+/*
+// On-board QSPI Flash
+#define EXTERNAL_FLASH_DEVICES W25Q16JVUXIQ
+#define EXTERNAL_FLASH_USE_QSPI
+
+*/
+
+/*
```

Comment:
see above, this comment is RAK specific and actually wrong for the ebyte module you are using.